addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1582,15 +1603,23 @@
                 width: 100%;
             }
             > div {
-                flex: 0 1 auto;
+                flex: 1 1 auto;
                 min-width: 0;
                 margin-top: $o-we-sidebar-content-field-spacing;
+
+                svg {
+                    margin: 0 auto;
+                }
             }
 
             &:not(.o_we_fw) {
                 flex-flow: row nowrap;
                 align-items: center;
 
+                &:not(we-input):not(.o_we_so_color_palette){
```

Comment:
Note that you can use `:not(A, B)` instead of :not(A):not(B)` now. Also there is a missing space here.